scope/scope.go

```diff
@@ -0,0 +1,76 @@
+package scope
+
+import (
+	"fmt"
+
+	"github.com/taskcluster/taskcluster-cli/extpoints"
+	"github.com/taskcluster/taskcluster-client-go"
+	"github.com/taskcluster/taskcluster-client-go/auth"
+)
+
+func init() {
+	extpoints.Register("scope", scope{})
+}
+
+type scope struct{}
+
+func (scope) ConfigOptions() map[string]extpoints.ConfigOption {
+	return nil
+}
+
+func (scope) Summary() string {
+	return "Expand the given scope set."
+}
+
+func usage() string {
+	return `Usage:
+  taskcluster SCOPE expand <scope>...
+
+### Expand
+This command returns an expanded copy of the given scope set, with scopes
+implied by any roles included. The given scope set is specified as a space
+separated list of scopes.
+`
+}
+
+func (scope) Usage() string {
+	return usage()
+}
+
+func (scope) Execute(context extpoints.Context) bool {
+	argv := context.Arguments
+
+	command := argv["SCOPE"].(string)
+	provider := extpoints.CommandProviders()[command]
+	if provider == nil {
+		panic(fmt.Sprintf("Unknown command: %s", command))
+	}
+
+	// Set credentials
+	authCreds := tcclient.Credentials(*context.Credentials)
+	myAuth := auth.New(&authCreds)
```

Comment:
Checkout: https://docs.taskcluster.net/reference/platform/auth/api-docs#expandScopes

No scopes, and by implication no credentials are required. I'm sure you can just create empty Auth with empty credentials and set Authenticate: false:
https://github.com/taskcluster/taskcluster-client-go/blob/master/auth/auth.go#L92